cocos/2d/CCSprite.cpp

```diff
@@ -152,9 +154,11 @@ bool Sprite::initWithTexture(Texture2D *texture)
     CCASSERT(texture != nullptr, "Invalid texture for sprite");
 
     Rect rect = Rect::ZERO;
-    rect.size = texture->getContentSize();
+    if (texture) {
+        rect.size = texture->getContentSize();
+    }
 
-    return initWithTexture(texture, rect);
+    return initWithTexture(texture, rect, false);
```

Comment:
The `rotated` default to `false`. You can refer to `virtual bool initWithTexture(Texture2D *texture, const Rect& rect);`